assets/src/settings-page/welcome.js

```diff
@@ -0,0 +1,98 @@
+							<path d="M8.92266 43.6992C8.42266 44.0992 7.52266 44.6992 6.72266 45.1992" stroke="#2459E7" strokeWidth="2" strokeLinecap="round" />
+							<path d="M7.42261 39.8984C5.92261 40.4984 2.82261 41.5984 1.92261 41.7984" stroke="#2459E7" strokeWidth="2" strokeLinecap="round" />
+							<path d="M3.92251 48.8992C4.80617 48.8992 5.52251 48.1829 5.52251 47.2992C5.52251 46.4156 4.80617 45.6992 3.92251 45.6992C3.03885 45.6992 2.32251 46.4156 2.32251 47.2992C2.32251 48.1829 3.03885 48.8992 3.92251 48.8992Z" fill="#2459E7" />
+							<path d="M60.1227 12.7C61.0064 12.7 61.7227 11.9837 61.7227 11.1C61.7227 10.2163 61.0064 9.5 60.1227 9.5C59.2391 9.5 58.5227 10.2163 58.5227 11.1C58.5227 11.9837 59.2391 12.7 60.1227 12.7Z" fill="#2459E7" />
+						</g>
+						<defs>
+							<clipPath id="welcome-svg-clip">
+								<rect width="60.8" height="50" fill="white" transform="translate(0.922607 0.398438)" />
+							</clipPath>
+						</defs>
+					</svg>
+
+				</div>
+
+				<div className="settings-welcome__body">
+					<h2>
+						{ pluginConfigured
+							? (
+								<>
+									{ __( 'AMP Settings Configured', 'amp' ) }
+
+									<svg width="25" height="25" viewBox="0 0 25 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+										<mask id="check-circle-mask" mask-type="alpha" maskUnits="userSpaceOnUse" x="2" y="2" width="21" height="21">
+											<path fillRule="evenodd" clipRule="evenodd" d="M12.7537 2.60938C7.23366 2.60938 2.75366 7.08938 2.75366 12.6094C2.75366 18.1294 7.23366 22.6094 12.7537 22.6094C18.2737 22.6094 22.7537 18.1294 22.7537 12.6094C22.7537 7.08938 18.2737 2.60938 12.7537 2.60938ZM12.7537 20.6094C8.34366 20.6094 4.75366 17.0194 4.75366 12.6094C4.75366 8.19938 8.34366 4.60938 12.7537 4.60938C17.1637 4.60938 20.7537 8.19938 20.7537 12.6094C20.7537 17.0194 17.1637 20.6094 12.7537 20.6094ZM10.7537 14.7794L17.3437 8.18937L18.7537 9.60938L10.7537 17.6094L6.75366 13.6094L8.16366 12.1994L10.7537 14.7794Z" fill="white" />
+										</mask>
+										<g mask="url(#check-circle-mask)">
+											<rect x="0.753662" y="0.609375" width="24" height="24" fill="#2459E7" />
+										</g>
+									</svg>
+								</>
+							)
+							: __( 'Configure AMP', 'amp' )
+						}
+
+					</h2>
+					<p>
+						{ __( 'The AMP configuration wizard can help you choose the best settings for your theme, plugins, and technical capabilities.', 'amp' ) }
+					</p>
+
+					{
+						onboardingWizardLink && (
+							<a className="components-button is-primary settings-welcome__button" href={ onboardingWizardLink } >
+								{ pluginConfigured ? __( 'Reopen Wizard', 'amp' ) : __( 'Open Wizard', 'amp' ) }
+							</a>
+						)
+
+					}
+					{
+						customizerLink && (
+							<a className="components-button is-link" href={ customizerLink } target="_blank" rel="noreferrer">
+								{ __( 'Visit Customizer', 'amp' ) }
```

Comment:
When in the middle of making changes in the Settings screen, should clicking on `Visit Customizer` link warn the user that changes have been made which have not been saved? The user can select e.g. 2019, then visit customizer to make changes on currently selected theme, then go back and save the. settings changes.

Reply:
We could do this very easily by having the customizer link open in the same tab. Then users would see the "Leave site?" warning if they have unsaved changes.

Reply:
Yes, let's do this. We need to warn the user because if they switch the Reader theme or mode then the Customizer link may not be current. So they need to save so that the up-to-date link is shown.

Reply:
Done in 13193a5